new-in/instantsearchjs/index.html

```diff
@@ -0,0 +1,34 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <!-- Do not use @2 in production, use a complete version like x.x.x, see website for latest version:
+      https://community.algolia.com/instantsearch.js/v2/getting-started.html
+    -->
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/instantsearch.js@2/dist/instantsearch.min.css">
+	  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/instantsearch.js@2/dist/instantsearch-theme-algolia.min.css">
+	  <link rel="stylesheet" href="./index.css">
+    <meta charset="utf-8">
+    <title>Algolia InstantSearch.js NEW IN</title>
+  </head>
+
+  <body>
+    <div class="container">
+      <details open>
+        <h1>Algolia <a href="https://community.algolia.com/instantsearch.js/">InstantSearch.js</a> template</h1>
+        <p>New in widget, such as the one described here: https://algolia.atlassian.net/wiki/spaces/SK/pages/577110177/New+In+Items.</p>
```

Comment:
this document should be added as the readme, in the same style as the other readmes

Reply:
I can't access the document anymore 😢 
Closing the PR as I can't help much here. Up to @sylvain-h to re-open it if it is still a hot topic.